src/subscription_manager/overrides.py

```diff
@@ -12,71 +12,78 @@
 
 # Module for manipulating content overrides
 
 
 class Overrides(object):
     def __init__(self):
-        self.cp_provider = inj.require(inj.CP_PROVIDER)
+        self.cp_provider: CPProvider = inj.require(inj.CP_PROVIDER)
 
-        self.cache = inj.require(inj.OVERRIDE_STATUS_CACHE)
+        self.cache: OverrideStatusCache = inj.require(inj.OVERRIDE_STATUS_CACHE)
         self.repo_lib = RepoActionInvoker(cache_only=True)
 
-    def get_overrides(self, consumer_uuid):
+    def get_overrides(self, consumer_uuid: str) -> List["Override"]:
         return self._build_from_json(self.cache.load_status(self._getuep(), consumer_uuid))
 
-    def add_overrides(self, consumer_uuid, overrides):
+    def add_overrides(self, consumer_uuid, overrides) -> List["Override"]:
         return self._build_from_json(self._getuep().setContentOverrides(consumer_uuid, self._add(overrides)))
 
-    def remove_overrides(self, consumer_uuid, overrides):
+    def remove_overrides(self, consumer_uuid, overrides) -> List["Override"]:
         return self._delete_overrides(consumer_uuid, self._remove(overrides))
 
-    def remove_all_overrides(self, consumer_uuid, repos):
+    def remove_all_overrides(self, consumer_uuid, repos) -> List["Override"]:
         return self._delete_overrides(consumer_uuid, self._remove_all(repos))
 
-    def update(self, overrides):
+    def update(self, overrides: List["Override"]) -> None:
         self.cache.server_status = [override.to_json() for override in overrides]
         self.cache.write_cache()
         self.repo_lib.update()
 
-    def _delete_overrides(self, consumer_uuid, override_data):
+    def _delete_overrides(self, consumer_uuid: str, override_data: List[dict]):
         return self._build_from_json(self._getuep().deleteContentOverrides(consumer_uuid, override_data))
 
-    def _add(self, overrides):
+    def _add(self, overrides: List) -> List[dict]:
         return [override.to_json() for override in overrides]
 
-    def _remove(self, overrides):
+    def _remove(self, overrides: List) -> List[dict]:
         return [{"contentLabel": override.repo_id, "name": override.name} for override in overrides]
 
-    def _remove_all(self, repos):
+    def _remove_all(self, repos: List[dict]) -> Optional[List[dict]]:
         if repos:
             return [{"contentLabel": repo} for repo in repos]
         else:
             return None
 
-    def _build_from_json(self, override_json):
+    def _build_from_json(self, override_json: List[dict]) -> List["Override"]:
         return [Override.from_json(override_dict) for override_dict in override_json]
 
-    def _getuep(self):
+    def _getuep(self) -> "UEPConnection":
         return self.cp_provider.get_consumer_auth_cp()
 
 
 class Override(object):
-    def __init__(self, repo_id, name, value=None):
-        self.repo_id = repo_id
-        self.name = name
-        self.value = value
+    def __init__(self, repo_id: str, name: str, value: Optional = None):
```

Comment:
i guess `Optional[object]`, just like it is set few lines below for `self.value`?